code/pagetypes/Forum.php

```diff
@@ -731,9 +731,13 @@ function BBTags() {
 	 *
 	 * @return Form Returns the post message form
 	 */
-	function PostMessageForm($addMode = false, $post = false) {
-		$thread = false;
+	function PostMessageForm($addMode = true, $post = false) {
```

Comment:
Agreed. I will look at creating a 0.8 branch tomorrow this can go against (there are a few other breaking change PRs that can all go towards a 0.8.0 release tomorrow perhaps.